package.json

```diff
@@ -30,7 +30,9 @@
     "@reduxjs/toolkit": "1.5.1",
     "axios": "0.21.1",
     "babel-preset-react-app": "10.0.0",
+    "bluebird": "^3.7.2",
```

Comment:
Please remove `^` from this file to maintain the exact version of the package to avoid problems with updates